samples/dotnet_core_managed_gateway/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO;
+using Microsoft.Azure.Devices.Gateway;
+
+namespace dotnet_core_managed_gateway
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                if (args.Length < 1 || File.Exists(args[0]) == false)
+                {
+                    Console.Error.WriteLine("Usage: managed-gateway <<path to json>>");
+                }
+                else
+                {
+                    var json_path = args[0];
+                    var gateway = GatewayInterop.CreateFromJson(json_path);
+
+                    Console.WriteLine(".NET Core Gateway is running. Press return to quit.");
+                    Console.ReadLine();
+
+                    GatewayInterop.Destroy(gateway);
+                }
+            }
+            catch (Exception ex)
+            {
+                Console.Error.WriteLine($"ERROR: {ex.ToString()}");
```

Comment:
Agree. Removing. Thanks!

---
In reply to: [105793981](https://github.com/Azure/azure-iot-gateway-sdk/pull/163#discussion_r105793981) [](ancestors = 105793981)